modules/standard/Sys.chpl

```diff
@@ -222,8 +257,22 @@ module Sys {
       this.complete();
       sys_init_sys_sockaddr_t(this);
     }
+    proc fill(host: c_string, port: uint(16), family: int):c_int {
```

Comment:
I'm not so sure what `fill` means? It's a new function, right? Maybe we can come up with a better name and add some documentation comments for it?

Reply:
I will add in the comments... for name 😅 am open to suggestions wasn't able to come up with one. The purpose of this function is to `fill` in details like `family`, `port`, and `address` into the `sys_sockaddr_t`

Reply:
what about `set` ?